.github/workflows/build.yml

```diff
@@ -2,25 +2,29 @@ name: Build
 on:
   pull_request:
     branches:
-    - master
+    - main
```

Comment:
We might want to do the branch rename separately?

Reply:
I was thinking that we do the repo move, branch rename, and then attempt to run and land this. Is there any value in separating it?

Reply:
I think that sequence would work.